src/app/globals.css

```diff
@@ -90,3 +90,12 @@
   text-decoration: underline;
   color: blue;
 }
+
+.primary-badge-outline {
+  border-color: RGB(173, 250, 29);
+  color: RGB(173, 250, 29);
+}
+
+.secondary-badge-outline {
+  background-color: RGB(82, 165, 0);
+}
```

Comment:
You should remove this and use tailwind instead to set css colors, we are using the themes with light mode and dark mode, if we start setting colors on the CSS file firectly it will break the theming